From 24ac19e1816ab2ec93360d346edd16e3f678bf47 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Zbigniew=20J=C4=99drzejewski-Szmek?= <zbyszek@in.waw.pl>
Date: Wed, 6 Apr 2022 11:56:05 +0200
Subject: [PATCH] hwdb: drop boilerplate about match patterns being unstable

We've had this text since the beginning, but in fact the patterns must be
stable in order for people to create local hwdb entries. And we support that
and can't change the match patterns without being very careful. So let's just
drop the text.

(cherry picked from commit adbe000e3df5f91572bf4a77a2acd84ff5b87320)

Related: #2087778
---
 hwdb.d/60-evdev.hwdb         | 5 -----
 hwdb.d/60-input-id.hwdb      | 5 -----
 hwdb.d/60-keyboard.hwdb      | 5 -----
 hwdb.d/60-sensor.hwdb        | 5 -----
 hwdb.d/70-mouse.hwdb         | 5 -----
 hwdb.d/70-pointingstick.hwdb | 4 ----
 6 files changed, 29 deletions(-)

diff --git a/hwdb.d/60-evdev.hwdb b/hwdb.d/60-evdev.hwdb
index 9fcb4a3ddf..c97cd13557 100644
--- a/hwdb.d/60-evdev.hwdb
+++ b/hwdb.d/60-evdev.hwdb
@@ -3,11 +3,6 @@
 # The lookup keys are composed in:
 #   60-evdev.rules
 #
-# Note: The format of the "evdev:" prefix match key is a contract between the
-# rules file and the hardware data, it might change in later revisions to
-# support more or better matches, it is not necessarily expected to be a stable
-# ABI.
-#
 # Match string formats:
 # evdev:<modalias>
 # evdev:name:<device name>:dmi:<dmi string>
diff --git a/hwdb.d/60-input-id.hwdb b/hwdb.d/60-input-id.hwdb
index 2d5681dea6..802ed9a1e7 100644
--- a/hwdb.d/60-input-id.hwdb
+++ b/hwdb.d/60-input-id.hwdb
@@ -3,11 +3,6 @@
 # The lookup keys are composed in:
 #   60-input-id.rules
 #
-# Note: The format of the "input-id:" prefix match key is a contract between
-# the rules file and the hardware data, it might change in later revisions to
-# support more or better matches, it is not necessarily expected to be a stable
-# ABI.
-#
 # Match string formats:
 # id-input:modalias:<modalias>
 #
diff --git a/hwdb.d/60-keyboard.hwdb b/hwdb.d/60-keyboard.hwdb
index c196a1fd64..7c830fb9b1 100644
--- a/hwdb.d/60-keyboard.hwdb
+++ b/hwdb.d/60-keyboard.hwdb
@@ -13,11 +13,6 @@
 # The lookup keys are composed in:
 #   60-evdev.rules
 #
-# Note: The format of the "evdev:" prefix match key is a contract between the
-# rules file and the hardware data, it might change in later revisions to
-# support more or better matches, it is not necessarily expected to be a stable
-# ABI.
-#
 # Supported hardware matches are:
 #  - Generic input devices match:
 #      evdev:input:bZZZZvYYYYpXXXXeWWWW-VVVV
diff --git a/hwdb.d/60-sensor.hwdb b/hwdb.d/60-sensor.hwdb
index ea278913bc..b54539bf53 100644
--- a/hwdb.d/60-sensor.hwdb
+++ b/hwdb.d/60-sensor.hwdb
@@ -3,11 +3,6 @@
 # The lookup keys are composed in:
 #   60-sensor.rules
 #
-# Note: The format of the "sensor:" prefix match key is a contract between the
-# rules file and the hardware data, it might change in later revisions to
-# support more or better matches, it is not necessarily expected to be a stable
-# ABI.
-#
 # Match string formats:
 # sensor:modalias:<parent modalias pattern>:dmi:<dmi pattern>
 #
diff --git a/hwdb.d/70-mouse.hwdb b/hwdb.d/70-mouse.hwdb
index 723d145e34..4084019bde 100644
--- a/hwdb.d/70-mouse.hwdb
+++ b/hwdb.d/70-mouse.hwdb
@@ -6,11 +6,6 @@
 # The lookup keys are composed in:
 #   70-mouse.rules
 #
-# Note: The format of the "mouse:" prefix match key is a contract between the
-# rules file and the hardware data, it might change in later revisions to
-# support more or better matches, it is not necessarily expected to be a stable
-# ABI.
-#
 # Match key format:
 # mouse:<subsystem>:v<vid>p<pid>:name:<name>:
 #
diff --git a/hwdb.d/70-pointingstick.hwdb b/hwdb.d/70-pointingstick.hwdb
index 6039119b6a..00f28c6034 100644
--- a/hwdb.d/70-pointingstick.hwdb
+++ b/hwdb.d/70-pointingstick.hwdb
@@ -7,10 +7,6 @@
 # The lookup keys are composed in:
 #   60-evdev.rules
 #
-# Note: The format of the "evdev:" prefix match key is a contract between the
-# rules file and the hardware data, it might change in later revisions to
-# support more or better matches, it is not necessarily a stable ABI.
-#
 # Supported hardware matches are:
 #  - Generic input devices match:
 #      evdev:input:bZZZZvYYYYpXXXXeWWWW-VVVV
